specs/fulu/das-core.md

```diff
@@ -191,7 +191,7 @@ Columns are grouped into custody groups. Nodes custodying a custody group MUST c
 
 A node *may* choose to custody and serve more than the minimum honesty requirement. Such a node explicitly advertises a number greater than `CUSTODY_REQUIREMENT` through the peer discovery mechanism, specifically by setting a higher value in the `custody_group_count` field within its ENR. This value can be increased up to `NUMBER_OF_CUSTODY_GROUPS`, indicating a super-full node.
 
-A node stores the custodied columns for the duration of the pruning period and responds to peer requests for samples on those columns.
+A node is required to automatically detect the custody groups based on any attached validators, without any input from the user. A node stores the custodied columns for the duration of the pruning period and responds to peer requests for samples on those columns. Finally, the node is expected to respond to changes in the amount of stake associated with any of the attached validators without a user input.
```

Comment:
i'd suggest adding these two additions as two notes, following this style:

https://github.com/ethereum/consensus-specs/blob/ff99bc03d6da29d9ef6e055bdb8500e1b2942f1e/specs/phase0/validator.md?plain=1#L309